### PR TITLE
SQL: Not equals operator new syntax tests

### PR DESCRIPTION
**Pull request by unrealwork**
##  Not equals operator new syntax tests
- [x] Added tests for new operator `<>` that equals to `!=`
- [x] Added auxiliary methods:
   - `setTags` to Series class
  -  `filterRows` to StringTable class, that accepts args form arguments

### CODE REVIEW

File: src/test/java/com/axibase/tsd/api/model/sql/StringTable.java [hunk 1]
```diff
@@ -6,9 +6,9 @@
 
 /**
  * @author Igor Shmagrinskiy
- *
- * Class for storing SQL result table in {@link String}
- * objects. It is using custom deserializer
+ *         <p>
```
Comment (Sild) [inline src/test/java/com/axibase/tsd/api/model/sql/StringTable.java]: what's this?

**Comment by Sild** [comment]
review done

File: src/test/java/com/axibase/tsd/api/model/sql/StringTable.java (hunk 1 above)
Comment (unrealwork) [inline src/test/java/com/axibase/tsd/api/model/sql/StringTable.java]: reorganaze code by IDEA

**Comment by Sild** [comment]
ok

File: src/test/java/com/axibase/tsd/api/method/sql/operators/SqlNotEqualsOperator.java [hunk 2]
```diff
@@ -39,40 +40,29 @@ public static void prepareData() {
...
         final String sqlQuery = "" +
                 "SELECT entity, value, datetime FROM 'sql-not-equals-syntax-metric'" +
                 "WHERE datetime <> '2016-06-03T09:23:00.000Z' AND entity = 'sql-not-equals-syntax-entity'";
 
-        List<List<String>> resultRows = executeQuery(sqlQuery)
-                .readEntity(StringTable.class)
-                .filterRows("datetime");
-
-        assertTrue("Result rows must be empty", resultRows.isEmpty());
+        executeQuery(sqlQuery)
```
Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/sql/operators/SqlNotEqualsOperator.java]: what you expecting here?

Comment (unrealwork) [inline src/test/java/com/axibase/tsd/api/method/sql/operators/SqlNotEqualsOperator.java]: exception

**Pull request merged**